canonical-web-and-design/ubuntu.com
# Package.json improvements

- Separate out `watch-css` from `watch-js`, to speed file regeneration
- Fix watching of JS so it actually works
- Use `$(hostname -I | awk '{print $1;}')` to generate a real IP for the server

QA
--

```
dotrun
```

Check the output IP address server address works.

Try:

```
touch touch static/sass/_global-settings.scss
touch static/js/third-party/prism.js
```

Check the first time it only built CSS, the second time only JS.
